bin/anthology/papers.py

```diff
@@ -357,6 +371,90 @@ def as_bibtex(self, concise=False):
         # Serialize it
         return bibtex_make_entry(bibkey, bibtype, entries)
 
+    def as_citeproc_json(self):
+        """Return a citation object suitable for CiteProcJSON."""
+        if self._citeproc_json is None:
+            json = {
+                "id": self.bibkey,
```

Comment:
I see you're just using a dictionary here, but `json` is also a module, and since it might get imported some day, I'd suggest using a different name for the object.